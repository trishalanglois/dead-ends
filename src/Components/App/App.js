import React from 'react';
import './App.scss';
import Header from '../Header/Header';
import Form from '../../Containers/Form/Form'

function App() {
  return (
    <>
      <Header />
      <Form />
    </>
  );
}

export default App;
