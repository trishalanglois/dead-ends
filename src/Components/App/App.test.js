import React from 'react';
import { render } from '@testing-library/react';
import App from './App';
import { shallow } from 'enzyme';


describe('App', () => {
  it('should match the snapshot', () => {
    let wrapper = shallow(<App />)

    expect(wrapper).toMatchSnapshot();
  })
})
