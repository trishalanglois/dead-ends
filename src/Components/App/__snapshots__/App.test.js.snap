// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match the snapshot 1`] = `
<Fragment>
  <Route
    component={[Function]}
    path="/"
  />
  <Switch>
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(Form)",
          "type": [Function],
        }
      }
      exact={true}
      path="/"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(LocationContainer)",
          "type": [Function],
        }
      }
      path="/locations"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(FavoritesContainer)",
          "type": [Function],
        }
      }
      path="/favorites"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(Wrapper)",
          "type": [Function],
        }
      }
      path="/location"
    />
    <Route
      component={[Function]}
    />
  </Switch>
</Fragment>
`;
